docs/reference/data-frames/apis/stop-transform.asciidoc

```diff
@@ -68,6 +71,11 @@ If this parameter is `false`, the request returns a `404` status code when there
 are no matches or only partial matches.
 --
 
+`force`::
+  (Optional, boolean) If `true`, resets the state of a failed {transform} to 
```

Comment:
I suggest changing the description of the "force" parameter as follows to be closer to what's in https://www.elastic.co/guide/en/elasticsearch/reference/master/ml-close-job.html :

>  Use to close a failed {transform} or to forcefully close a {transform} that has not responded to its initial stop request.